src/content-helper/common/settings/provider.tsx

```diff
@@ -59,13 +59,13 @@ const SettingsContext = createContext<SettingsContextType<Settings>>( {
  * // Using the useSettings hook with a specific type
  * const { settings, setSettings } = useSettings<SidebarSettings>();
  */
-export const useSettings = <T = Settings>(): SettingsContextType<T> => {
+export function useSettings<T = Settings>(): SettingsContextType<T> {
```

Comment:
Changed this to `function` instead of a arrow function, since the syntax was somehow borking my IDE's syntax highlighting. 

This has the same effect, and I still get pretty colors on this file :)